pkg/controller/plan/kubevirt.go

```diff
@@ -834,6 +834,56 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 	return
 }
 
+func (r *KubeVirt) EnsureVirtV2VPVCStatus() (ready bool, err error) {
+	pvcs := &core.PersistentVolumeClaimList{}
+	pvcLabels := map[string]string{
+		"migration": r.Migration.Name,
+		"ova":       "nfs-pvc",
+	}
+
+	err = r.Destination.Client.List(
+		context.TODO(),
+		pvcs,
+		&client.ListOptions{
+			LabelSelector: labels.SelectorFromSet(pvcLabels),
+			Namespace:     r.Plan.Spec.TargetNamespace,
+		},
+	)
+	if err != nil {
+		return false, err
+	}
+
+	if len(pvcs.Items) == 0 {
+		return
+	}
+
+	pvc := &core.PersistentVolumeClaim{}
+	if len(pvcs.Items) > 1 {
+		for _, pvcVirtV2v := range pvcs.Items {
+			if pvcVirtV2v.CreationTimestamp.Time.After(r.Migration.CreationTimestamp.Time) {
+				pvc = &pvcVirtV2v
```

Comment:
what are we trying to achieve here? the last PVC that was made?

Reply:
yes, for any case if we will have a list with more then one PVC filter the correct one by date, although it should not happen since we filter the list by migration name so we should have only one PVC at all times.

Reply:
but since you lookup for the all PVCs of the migration and not of the VM you may get a PVC that isn't related to your VM.
Also, it will always pick the last PVC that was made after the migration did? you don't compare the creation time of the PVCs between them.

`it should not happen since we filter the list by migration name so we should have only one PVC at all times.` - i didn't get that part.

Reply:
yes the vmID was a good point so I've added that label to the PVC (I comment on that in one of the other threats:))
and after that we really will have only one PVC ( since we creating only one per vm per migration) and in case we don't - if some leftovers will be cough up - we will pick up the last one - which mean the newer one created.

Reply:
we don't support multiple disks VM when importing from OVA?

Reply:
we do, this is the PVC that we used as nfs to get the ova/ovf source for the migration, not the disks of the VM. so from this type we have only one per migration attached to the virt-v2v pod.